src/v/json/json.cc

```diff
@@ -11,48 +11,38 @@
 
 namespace json {
 
-void rjson_serialize(rapidjson::Writer<rapidjson::StringBuffer>& w, short v) {
-    w.Int(v);
-}
+void rjson_serialize(json::Writer<json::StringBuffer>& w, short v) { w.Int(v); }
 
-void rjson_serialize(rapidjson::Writer<rapidjson::StringBuffer>& w, bool v) {
-    w.Bool(v);
-}
+void rjson_serialize(json::Writer<json::StringBuffer>& w, bool v) { w.Bool(v); }
```

Comment:
nit/clarification: Should the prefix to these function names now be `json_*` instead of `rjson_*`?

Reply:
I'd like to hide the entire serialise and parse interfaces behind [tag_invoke](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2019/p1895r0.pdf) - but I'm saving that churn for another day.

Reply:
Yeah there's no need to do that now. Good stuff!